plugins/fluent-bit/src/plugin.cpp

```diff
@@ -131,6 +131,7 @@ auto visit(auto f, const msgpack_object& object) {
     case MSGPACK_OBJECT_EXT:
       return f(object.via.ext);
   }
+  return f(ec::unspecified);
```

Comment:
Why are dispatching to a CAF error code? There's no CAF at all nearby.

Reply:
We're dispatching `tenzir::ec` values as to catch them in upstream methods that call `msgpack::visit()`, such as `add()` (pretty much the only method that calls `msgpack::visit()` anyway).

Would you prefer a semantically cleaner type, such as `std::monostate`?

Reply:
I don't want to bikeshed on this too much, but anything that smells like an error structure from `std*` would feel more natural.

Reply:
I replaced the error type with an `std::runtime_error`.

Reply:
Basically, yes, a non-`msgPack` tag type.

Is there one you'd recommend?

Reply:
I implemented `unknown_msgpack_type`.